### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/react_static-list-of-todos/

### CODE REVIEW

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <>
+    <section className="TodoList">
+      {todos.map(todo => (<TodoInfo todo={todo} key={todo.id} />))}
+    </section>
+  </>
```
Comment (DarkMistyRoom) [inline src/components/TodoList/TodoList.tsx:13]: redundant Fragment

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <>
+      <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
+        <h2 className="TodoInfo__title">{todo.title}</h2>
+        {todo.user && <UserInfo user={todo.user} />}
+      </article>
+    </>
```
Comment (DarkMistyRoom) [inline src/components/TodoInfo/TodoInfo.tsx:15]: redundant Fragment

File: src/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
 );
```
Comment (DarkMistyRoom) [inline src/components/UserInfo/UserInfo.tsx:13]: redundant Fragment

File: src/components/TodoInfo/TodoInfo.tsx [hunk 4]
```diff
@@ -1,4 +1,17 @@
...
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <>
+      <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
```
Comment (DarkMistyRoom) [inline src/components/TodoInfo/TodoInfo.tsx:11]: you can use classNames to simplify the syntax

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by Virass — APPROVED** [review]

**Pull request closed without merging**